bedrock/press/templates/press/speaker-request.html

```diff
@@ -232,7 +232,7 @@ <h2>
     <fieldset class="mzp-c-field-set">
       <legend class="mzp-c-form-subtitle">Additional Information</legend>
 
-      <div class="mzp-c-field {% if form.sr_attachment.errors %}mzp-is-error{% endif %}" id="sr_attachment">
+      <div class="mzp-c-field {% if form.sr_attachment.errors %}mzp-is-error{% endif %}">
```

Comment:
FYI: The (same) ID is on the rendered field_with_attrs — and since label targets resolve for the first element found, be it a valid form input or an N/A non-"labellable" element, this dupe has to be removed for the label to actually work with the intended field.